### PR TITLE
Add PidState message

### PR DESCRIPTION
**Pull request by paulbovbel**

### CODE REVIEW

**Comment by paulbovbel** [comment]
Ping

**Comment by davetcoleman** [comment]
lgtm +1

File: control_msgs/msg/PidState.msg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Header header
+float64 error
+float64 error_dot
+float64 dt
```
Comment (adolfo-rt) [inline control_msgs/msg/PidState.msg]: Would it make more sense to make this a duration?.
Also, how about renaming `dt` to `timestep`?

**Comment by adolfo-rt** [comment]
Looks mostly fine. Few inline comments.

**Comment by paulbovbel** [comment]
Updated, ping for review

File: control_msgs/msg/PidState.msg (hunk 1 above)
Comment (paulbovbel) [inline control_msgs/msg/PidState.msg]: agreed!

**Comment by adolfo-rt** [comment]
LGTM

**Pull request merged**